### DISCUSSION TITLE
Cant find files

### DISCUSSION THREAD

**Discussion by Hadeaen** [tteck/Proxmox#2512] [Help Needed]
Hi,

Im kinda new in this and i was installing with this helper scripts and i can find directories to edit the files.
In my case i try to install Homepage, but cant find configurations files. Whenever i try to locate the files it is not where it should be.

Could somebody please explain where or how to find files location for Homepage LXC.

Thank you.

**Comment by tteck** [comment]
First, did you read the instructions?

**Comment by Hadeaen** [comment]
Sorry, did now, find the solution.

Sorry for trouble.
Thank you
